chunkset_tpl.h

```diff
@@ -176,7 +176,18 @@ Z_INTERNAL uint8_t* CHUNKMEMSET(uint8_t *out, unsigned dist, unsigned len) {
 }
 
 Z_INTERNAL uint8_t* CHUNKMEMSET_SAFE(uint8_t *out, unsigned dist, unsigned len, unsigned left) {
+#ifdef UNALIGNED64_OK
+    static const uint32_t align_mask = 3;
+#else
+    static const uint32_t align_mask = 7;
+#endif
```

Comment:
`align_mask` should only be used when:
1. UNALIGNED64_OK is undefined
2. UNALIGNED_OK and UNALIGNED64_OK are both undefined